### PR TITLE
Gherkin for disabling tektonhub integration in pipeline builder

### PR DESCRIPTION
**Pull request by sanketpathak**
Epic: https://issues.redhat.com/browse/ODC-6377
Story: https://issues.redhat.com/browse/ODC-6428

Acceptance criteria:

- Admins should be able to shut off integration with Tekton Hub

**Checks required for approving Epic gherkin scripts PR:**

- [x] Add @epic-number to the scenarios or feature file [e.g: @odc-xxx]
- [x] Add @to-do for automation possible scenarios
- [x] Add @regression or @smoke based on the importance of the scenario
- [x] Update the test scenarios count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)
- [x] Check for the linter issues by executing `yarn run gherkin-lint` on frontend folder [Skip epic number tags related linter issues]

### CODE REVIEW

File: frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature [hunk 1]
```diff
@@ -332,3 +332,19 @@ Feature: Create the pipeline from builder page
               And user clicks on Create button
               And user clicks on Logs tab in PipelineRun details page
              Then user will be able to see the output in sum and multiply task
+
+
+        @regression @manual @odc-6377
+        Scenario: Disable Tektonhub integrations in the pipeline builder : P-02-TC22
```
Comment (karthikjeeyar) [inline frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature:338]:
```suggestion
        Scenario: Disable Tektonhub integration in the pipeline builder : P-02-TC22
```

File: frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature [hunk 2]
```diff
@@ -332,3 +332,19 @@ Feature: Create the pipeline from builder page
...
+            Given user is at Search page
+              And user searches 'TektonConfig' in Resources dropdown
+              And user selects config with apiVersion operator.openshift.io/v1 option from Resources dropdown
+              And user clicks on "cluster" Name in Consoles
```
Comment (karthikjeeyar) [inline frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature:342]: The name of the tektonConfig CR will be `config`
image

File: frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature (hunk 1 above)
Comment (sanketpathak) [reply frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature]: Updated

File: frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature (hunk 2 above)
Comment (sanketpathak) [reply frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature]: Updated

**Comment by sanketpathak** [comment]
This PR only affects our e2e tests.

/label qe-approved
/label px-approved
/label docs-approved

**Pull request merged**